### PR TITLE
OF-1037: Split Bookmarks from ClientControl.

### PR DESCRIPTION
**Pull request by guusdk**

### CODE REVIEW

**Comment by sco0ter** [comment]
Is that an admin UI for XEP-0048?

**Comment by guusdk** [comment]
Yup - to the extend that it was already supported by ClientControl. I did not even look at the implementation, I simply splitted the existing plugin in two. See the JIRA issue for background / context.

**Comment by deleolajide** [comment]
+1

File: src/plugins/bookmarks/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <!-- Main plugin class -->
+    <class>org.igniterealtime.openfire.plugin.BookmarksPlugin</class>
+    
+    <!-- Plugin meta-data -->
+    <name>Bookmarks</name>
+    <description>Allows clients to store URL and group chat bookmarks (XEP-0048)</description>
+    <author>Ignite Realtime</author>
+    <version>1.0.0</version>
+    <date>1/11/2015</date>
+    <minServerVersion>3.11.0 alpha</minServerVersion>
```
Comment (tevans) [inline src/plugins/bookmarks/plugin.xml]: Note this version conflicts with the changes proposed in #506

**Comment by tevans** [comment]
+1, but might need a tweak to sync minServerVersion

**Comment by guusdk** [comment]
I merged #506 and rebased.

**Pull request merged**